java/src/main/java/lesson04/README.md

```diff
@@ -45,7 +46,7 @@ public static void main(String[] args) {
 And add this instruction to `sayHello` method after starting the span:
```

Comment:
Do we still need exit in L42, given that we're not using maven to run an longer?